crates/modelardb_storage/src/parser.rs

```diff
@@ -313,137 +486,174 @@ impl Dialect for ModelarDbDialect {
     }
 
     /// Check if the next tokens are CREATE MODEL TABLE, if so, attempt to parse the token stream as
```

Comment:
Maybe rewrite this doc comment so it also mentions `INCLUDE`.

Reply:
Fixed.